### PR TITLE
Update NZBVortex version

### PR DESCRIPTION
**Pull request by ptb**

### CODE REVIEW

File: Casks/nzbvortex.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask 'nzbvortex' do
-  version '2.11.10'
+  version '3.2.3'
   sha256 'bef3d0a6297bb4fff98361d4108061b7fff0c8d6ab1f2e2d88a56e67d9fa00b8'
```
Comment (vitorgalvao) [inline Casks/nzbvortex.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

Leave it, though, as there are other corrections needed on the cask, and will be making them in a different PR. Thank you for the contribution and bringing the updated version to our attention.

**Comment by ptb** [comment]
I think what happened is that the version was missed when it was last updated. The current download for v3.2.3 matches the sha256 listed.

**Comment by vitorgalvao** [comment]
> The current download for v3.2.3 matches the sha256 listed.

[It doesn’t](https://github.com/caskroom/homebrew-cask/pull/18808/files).

**Comment by vitorgalvao** [comment]
Even so, like I said [many other corrections were needed](https://github.com/caskroom/homebrew-cask/pull/18808/files).
